### ISSUE TITLE
models里面没必要做logits=self.softmax(logits)

### ISSUE THREAD

**Issue by leixingzhi7** [kitsch231/pytorch_fake_news_Classification_mml#7]
models里面没必要做logits=self.softmax(logits)，本身输出就是logits，后面用的是CrossEntropyLoss，里面会做softmax，虽然结果不影响，但是这步骤确实是多余了

**Comment by AirPlanBird** [comment]
我也想问

**Comment by kitsch231** [comment]
具体原因忘记了，可能是我当时尝试了很多的loss函数和模型结构来改进精度，不过目前没有其他影响。
